lib/std/compress/flate/CircularBuffer.zig

```diff
@@ -0,0 +1,234 @@
+/// 64K buffer of uncompressed data created in inflate (decompression). Has enough
```

Comment:
This should either be a container level doc comment (`//!`) or a regular comment.

Reply:
Thanks, sorry I didn't understand the difference.